### ISSUE TITLE
Accessibility: usage of <i> should be semantic

### ISSUE THREAD

**Issue by K-T-O** [CJSCommonPlatform/govuk_single_page_pdk#106]
Pa11y warns about the following issue:

- WCAG2AA.Principle1.Guideline1_3.1_3_1.H49.I

Warning: Semantic markup should be used to mark emphasised or special text so that it can be programmatically determined.

This occurs for example in gov-notification.

**Comment by fabiotisci** [comment]
could you paste how you have implemented the component? It's has multiple implementations and it is difficult to know what the problem could be otherwise

**Comment by K-T-O** [comment]
It exists for example in:
http://cjscommonplatform.github.io/govuk_single_page_pdk/#/components/banners
it's often used as an **i**con. 
While it has a specific semantic meaning in HTML5 and this is what pa11y warns about (it's not an error just a warning).

**Comment by fabiotisci** [comment]
@trevorsaint any input about this?

**Comment by trevorsaint** [comment]
@fabiotisci I would not worry so much about this particular issue. It's presentational and we have the following attributes applied:

role="presentation" aria-hidden="true"

So all is fine. If it is a really big issue we could always change it to a span. But lets leave it as it is.
